orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/DestroyServerGroupStage.groovy

```diff
@@ -40,11 +42,14 @@ class DestroyServerGroupStage extends TargetServerGroupLinearStageSupport {
 
   @Override
   public List<Step> buildSteps(Stage stage) {
-    injectBefore(stage, "disableServerGroup", disableServerGroupStage, stage.context)
```

Comment:
This previous behavior worked when the DestroyStage only targeted a single region.

With multiple regions, the subsequent child destroy stages were unable to find a DetermineTargetServerGroup stage and failed. 

May or may not be related to:
```
if (stage.parentStageId) {
      // We only want to determine the target server groups once per stage, so only inject if this is the root stage,
      // i.e. the one the user configured.
      // This may become a bad assumption, or a limiting one, in that we cannot inject a dynamic stage ourselves
      // as part of some other stage that is not itself injecting a determineTargetReferences stage.
      return
    }
```

Although it's a smidge of duplication, I actually prefer what the resulting stage and task graph looks like:

![image](https://cloud.githubusercontent.com/assets/388652/12841469/c65da93e-cba0-11e5-9d94-ccc9c471da86.png)